bullet_train-super_scaffolding/lib/scaffolding/script.rb

```diff
@@ -50,6 +50,20 @@ def standard_protip
   puts "If you do that, you can reset to your last commit state by using `git checkout .` and `git clean -d -f` ."
 end
 
+def git_status
+  `git status`.split("\n")
+end
+
+def has_untracked_files?(status_lines)
+  status_lines.include?("Untracked files:")
+end
+
+# All untracked files begin with a tab (i.e. - "\tapp/models/model.rb").
+def get_untracked_files(status_lines)
+  idx = status_lines.index("Untracked files:")
+  status_lines[idx..].select { |lines| lines.match?(/\t/) }
+end
+
```

Comment:
It looks like there's a way to get just the untracked files without needing the parsing https://stackoverflow.com/a/11024039

What if we did something like this?

```suggestion
def untracked_files_from_git
  `git ls-files --other --directory --exclude-standard`.split("\n")
end

```

Reply:
I definitely like using `git` directly better.
For some reason the command is returning ignored directories for me?
```
> git ls-files --other --directory --exclude-standard
.knapsack_pro/
.ruby-lsp/
app/models/project.rb
db/migrate/20231009090516_create_projects.rb
test/factories/projects.rb
test/models/project_test.rb
tmp/storage/
```

I think we can just get away with this:
```
> git ls-files --other --exclude-standard
app/models/project.rb
db/migrate/20231009090516_create_projects.rb
test/factories/projects.rb
test/models/project_test.rb
```

Just pushed the changes :+1: